### PR TITLE
Copying files on OSX fails. Closes #9.

### LINKED ISSUE #9: Copying files on OSX fails
The result of running build.sh on OSX:

```
Copying extra plugins...

usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file ... target_directory

64
undefined
```

### PR DESCRIPTION
**Pull request by wwalc**
A dummy workaround to deal with different cp implementation on OSX (Darwin).

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -93,8 +93,11 @@ cd ../..
 
 echo ""
 echo "Copying extra plugins..."
-cp -r plugins/* ckeditor/plugins/
-
+if [ "$(uname)" == "Darwin" ]; then
+	cp -R plugins/ ckeditor/plugins/
```
Comment (vokiel) [inline build.sh]: Man for Linux cp (GNU coreutils) 8.2 shows:
```
-R, -r, --recursive
              copy directories recursively
```
So it should work with `-R` for both OSX and Linux.

Comment (wwalc) [inline build.sh]: Well, someone has to check it on Linux and Win (cc @mlewand) to confirm. I did not want to mess up with the old command that used to work there.

Comment (vokiel) [inline build.sh]: `cp -R` works fine on Linux, tested on
* cp (GNU coreutils) 5.97 (CentOS 5-11.el5)
* cp (GNU coreutils) 8.25 (Linux Mint 18.1 / Ubuntu 16.04 LTS)

Comment (mlewand) [inline build.sh]: For Win is the same as for Ubuntu, so `--help` shows me:
```
  -R, -r, --recursive          copy directories recursively
```

Comment (mlewand) [inline build.sh]: Also checked git bash for windows and `-R` is also supported there.

I'm +1 for simplifying it to `-R` for all platforms.

Comment (vokiel) [inline build.sh]: Another thing, on original version there were `cp -r plugins/* ckeditor/plugins/` for OSX there is `-cp -R plugins/ ckeditor/plugins/` (no star) which I think works differently (at leas on Linux).
Example:
```$ tree
├── dest
└── source
    ├── s1
    │   └── file_1.txt
    └── s2
        └── file_2.txt
```

With star

```
$ cp -R source/* dest/
$ tree
├── dest
│   ├── s1
│   │   └── file_1.txt
│   └── s2
│       └── file_2.txt
└── source
    ├── s1
    │   └── file_1.txt
    └── s2
        └── file_2.txt
```

Without star

```
$ cp -R source/ dest/
$ tree
├── dest
│   └── source
│       ├── s1
│       │   └── file_1.txt
│       └── s2
│           └── file_2.txt
└── source
    ├── s1
    │   └── file_1.txt
    └── s2
        └── file_2.txt
```

Comment (wwalc) [inline build.sh]: On OSX the following two commands give equal results:

```
cp -R source/* dest/source/
cp -R source/ dest/source/
```
that was the reason why for OSX removed the superfluous asterisk (note that we're copying into a destination folder with the same name). I will add it back then.

**Comment by f1ames** [comment]
See https://github.com/ckeditor/ckeditor4-presets/issues/9#issuecomment-679177434.

**Pull request closed without merging**